packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -1014,9 +1016,9 @@ export default class InternalModel {
     }
   }
 
-  notifyBelongsToChange(key, record) {
+  notifyBelongsToChange(key: string) {
     if (this.hasRecord) {
-      this._record.notifyBelongsToChange(key, record);
+      this._record.notifyBelongsToChange(key, this._record);
```

Comment:
I don't think we need to pass in the second argument here, and in the case that we did, it would probably not be `this._record` but rather the value being set. Though that code path seems to have been removed, so we can just not pass the second arg